File: C Programming/Competitive Programming/Coding Challenges 5/program4.c
// 4. Write a program which accepts N from user and print all odd numbers up to N.
// Input : 18
// Output : 1 3 5 7 9 11 13

#include<stdio.h>

void OddDisplay(int iNo)
{
    int i = 0;
    if (iNo<0)
    {
        iNo=-iNo;
    }
    printf("Output is: ");
    for (i = 1; i <= iNo; i++)
    {
    if (i>13)
    {
        break;
    }
      if (i%2!= 0)
      {
        printf("%d\t",i);
      }
      
    }
    
    
}

int main()
{
    int iValue =0;

    printf("Enter Number : ");
    scanf("%d",&iValue);

    OddDisplay(iValue);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 12/program4.java
// 4.Write a program which accept number from user and display its table.
// Input : 2
// Output : 2 4 6 8 10 12 14 16 18 20
// Input : 5
// Output : 5 10 15 20 25 30 35 40 45 50
// Input : -5
// Output : 5 10 15 20 25 30 35 40 45 50

import java.util.Scanner;

class program4
{
    static void Table(int iNo)
    {
        if (iNo < 0)
        {
            iNo = -iNo;
        }

        int i = 1;

        System.out.print("Output is: ");

        do
        {
            System.out.print(" " + (iNo * i) + "\t");
            i++;
        }
        while (i <= 10);
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("Enter number : ");
        iValue = sobj.nextInt();

        Table(iValue);

        sobj.close();
    }
}